scrapper.py

```diff
@@ -25,39 +36,135 @@ class Crawler:
     """
     Crawler implementation
     """
+
     def __init__(self, seed_urls, max_articles: int):
-        pass
+        self.seed_urls = seed_urls
+        self.total_max_articles = max_articles
+        self.urls = []
 
     def _extract_url(self, article_bs):
-        pass
+        cells = article_bs.find_all("div", class_='col-lg-3 col-sm-6')
+        for cell in cells:
+            if len(self.urls) >= self.total_max_articles:
+                break
+            link = cell.find('a')
+            self.urls.append('https://pravdasevera.ru' + link['href'])
 
     def find_articles(self):
         """
         Finds articles
         """
-        pass
+        for seed_url in self.seed_urls:
+            if len(self.urls) >= self.total_max_articles:
```

Comment:
you have two same conditions in two functions - can we use only one condition?

Reply:
Yes